privval/file_test.go

```diff
@@ -300,8 +276,69 @@ func TestDifferByTimestamp(t *testing.T) {
 	}
 }
 
+func TestVoteExtensionsAreAlwaysSigned(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	privVal, _, _ := newTestFilePV(t)
```

Comment:
Let's use this function in places where this same logic exists in these tests.

Reply:
See above